moped-database/views/project_list_view.sql

```diff
@@ -82,6 +81,45 @@ moped_proj_components_subtypes AS (
     LEFT JOIN moped_components AS mc ON mpc.component_id = mc.component_id
     WHERE mpc.is_deleted = false
     GROUP BY mpc.project_id
+),
+
+project_district_association AS (
+
+    WITH
+    project_council_district_map AS (
+        SELECT DISTINCT
+            project_id,
+            unnest(council_districts) AS council_district
+        FROM project_geography
+    ),
+
+    parent_child_project_map AS (
+        SELECT
+            parent_projects.project_id,
+            unnest(
+                ARRAY[parent_projects.project_id]
+                || array_agg(child_projects.project_id)
+            ) AS self_and_children_project_ids
```

Comment:
This little trickery is constructing an array of all a given project's children id + its own id, making it a set of all IDs involved in project family.

Reply:
very cool! TIL